recipes/ragtag/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "RagTag" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f1c5dfdb05efddc0e3fc04f512e764e6d6ab3d7c7c796c8ca42f6ef50d5120a7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+
+requirements:
+  host:
+    - intervaltree
+    - numpy
+    - pip
+    - pysam
+    - python >3
+  run:
+    - intervaltree
+    - numpy
+    - pysam
+    - python >3
+
+test:
+  imports:
+    - ragtag_utilities
+
```

Comment:
You should also test for the Python scripts here as well, e.g.:
```
test:
  commands:
    - ragtag.py --help
```

Reply:
fixed in the latest commit - thanks